draft-birkholz-cose-receipts-ccf-profile.md

```diff
@@ -91,10 +91,10 @@ This documents extends the verifiable data structure registry of {{-cose-receipt
 
 | Name | Value | Description | Reference
 |---
-|CCF_LEDGER_SHA256 | TBD_1 (requested assignment 2) | Historical transaction ledgers, such as the CCF ledger | This document
+|CCF_LEDGER_SHA256 | TBD_1 (requested assignment 2) | Historical transaction ledgers, such as the CCF ledger | RFCthis
 {: #verifiable-data-structure-values align="left" title="Verifiable Data Structure Algorithms"}
 
-This document defines inclusion proofs for CCF ledgers. Verifiers MUST reject all other proof types
+This document defines inclusion proofs for CCF ledgers. Corresponding CCF Verifiers MUST reject proof types they do not support.
```

Comment:
@JAG-UK is quite right that this is a better wording, getting proofs from multiple ledgers definitely must be possible, and should be encouraged if anything.